index.js

```diff
@@ -286,3 +312,50 @@ function toBuffer (chunk, encoding) {
     ? Buffer.from(chunk, encoding)
     : chunk
 }
+
+/**
+ * Most browsers send "br" (brotli) as the last value in
+ * in the 'Accept-Encoding' header which causes it to be
+ * deprioritized according to the spec.
+ *
+ * This is typically not what end users actually want so here
+ * we force the "br" (brotli) value to first in the list so that
+ * it will get properly prioritized and used.
+ *
+ * It's worth noting that although this is not "spec compliant",
+ * we belive it follows a well-established convention.
+ *
+ * @private
+ */
+function prioritize (str) {
+  return str
```

Comment:
```suggestion
  if(str == undefined) {
    return undefined
  }
  return str
```

Reply:
Maybe I'm missing something here, but this looks redundant and doesn't do anything. @bloodyowl Would you mind explaining this suggestion?

Reply:
`str == undefined` would include `null` as well and return undefined instead of null

Reply:
That prevents trying to split `null` or `undefined` if the header isn't sent